platformio.ini

```diff
@@ -272,25 +303,21 @@ build_flags     = -DUSE_PSRAM=1
 ; Libs needed to link with a TTGO Module
 [ttgo_deps]
 lib_deps        = https://github.com/Xinyuan-LilyGO/TTGO-T-Display
-                  TFT_eSPI
-
-; Build flags for enabling high precision Unity double
-[unity_double_flags]
-build_flags     = -DUNITY_INCLUDE_DOUBLE
-                  -DUNITY_DOUBLE_PRECISION=1e-12    ; Make doubles real 8 byte doubles
+                  bodmer/TFT_eSPI @ ^2.5.43
 
 ; ================================================================
 ; Basic environment section, automatically inherited by all others
 ;
 
 [env]
-platform        = platformio/espressif32@^6.3.0
+platform        = platformio/espressif32 @ ^6.12.0
 framework       = arduino
 build_type      = release
 build_unflags   = -std=gnu++11
 lib_extra_dirs  = ${PROJECT_DIR}/lib
 monitor_filters = esp32_exception_decoder
-extra_scripts   = pre:tools/bake_site.py
+extra_scripts   = pre:install_intelhex.py
```

Comment:
Again, if it turns out install_intelhex.py is no longer necessary, then neither is this.